### DISCUSSION TITLE
Isar database backup locally and restore

### DISCUSSION THREAD

**Discussion by andrySD** [isar/isar#875] [Help]
I want to know if there is a way by which I can export all the database of the Isar into a backup and then restore it so if a user wants to uninstall the app the user can backup the data locally into a location like sdcard or phone storage and after reinstalling they can restore from backup all the database and can use it again. So what can be the ways by which it can be done?

I see [this ](https://github.com/hivedb/hive/issues/629)post about Hive and I copy and paste becouse i had the same questions, but with Isar.

**Comment by spatialbits** [comment]
Pretty sure this has been discussed elsewhere already, but you can just export to json and later import from json. You'll likely need to account for schema changes in the meantime, however.

**Reply by andrySD** [reply]
I could not find any discution about this topic on internet. Anywhay, how can i export the Isar data base to .json?
